### PR TITLE
BigAnimal: Azure AD and Okta IdP topics

### PR DESCRIPTION
**Pull request by drothery-edb**
## What Changed?

Added topics for setting up Azure AD and OKTA as your Idp

### CODE REVIEW

File: product_docs/docs/biganimal/release/getting_started/identity_provider/azure_ad.mdx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+   
+      Where &lt;assertion_path> is http://schemas.xmlsoap.org/ws/2005/05/identity/claims.
+
+   1. Under **SAML Signing Certificate**, select **Download** for the Base64-encoded certificate.
+   1. Copy the **Login URL** value.
+1. From the left navigation pane, Select **Properties**. On the Properties page, if you don't want all of the users in Azure AD to use BigAnimal, answer `No` for the **Assignment required?** question. Otherwise, answer `Yes` and assign the specified users to this Enterprise Application explicitly on the Users and Groups page.
+1. On the **Setup Config** tab on the Set Up Identity Provider page in BigAnimal: 
+   1. Paste the Login URL value you copied from the Single sign-on page in Azure AD as the **Single Sign-On URL**.
+   1. For **Identity Provider Signature Certificate**, upload the Base64-encoded certificate downloaded from Azure.
+   1. Select the appropriate method for **Request Binding**. Azure AD supports HTTP POST and HTTP Redirect.
```
Comment (xugy99) [inline product_docs/docs/biganimal/release/getting_started/identity_provider/azure_ad.mdx:57]: Azure AD supports HTTP-Post and Hybrid 

(HTTP-Redirect choice is not supported by Azure AD per our test).

File: product_docs/docs/biganimal/release/getting_started/identity_provider/azure_ad.mdx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      Where &lt;assertion_path> is http://schemas.xmlsoap.org/ws/2005/05/identity/claims.
+
+   1. Under **SAML Signing Certificate**, select **Download** for the Base64-encoded certificate.
+   1. Copy the **Login URL** value.
+1. From the left navigation pane, Select **Properties**. On the Properties page, if you don't want all of the users in Azure AD to use BigAnimal, answer `No` for the **Assignment required?** question. Otherwise, answer `Yes` and assign the specified users to this Enterprise Application explicitly on the Users and Groups page.
+1. On the **Setup Config** tab on the Set Up Identity Provider page in BigAnimal: 
+   1. Paste the Login URL value you copied from the Single sign-on page in Azure AD as the **Single Sign-On URL**.
+   1. For **Identity Provider Signature Certificate**, upload the Base64-encoded certificate downloaded from Azure.
+   1. Select the appropriate method for **Request Binding**. Azure AD supports HTTP POST and HTTP Redirect. 
+   1. Select **rsa-sha256** for the **Response Signature Algorithm**.
```
Comment (xugy99) [inline product_docs/docs/biganimal/release/getting_started/identity_provider/azure_ad.mdx:58]: rsh-sha1 is also supported by Azure AD

File: product_docs/docs/biganimal/release/getting_started/identity_provider/okta.mdx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+   1. On the **Assignments** tab on the Applications page, select **Assign** to assign people or groups to the newly created application. If you need to sign into BigAnimal, be sure to assign yourself.
+   1.  On the **Sign On** tab, select **View SAML setup instructions** to open a tab with instructions for your application. 
+       1. Copy from the Identity Provider Single Sign-on URL from step 1 of the SAML setup instructions.
+       1. Select **Download certificate** in step 3 of the instructions.
+
+   1. In BigAnimal, on the **Setup Config** tab on the Set Up Identity Provider page: 
+      1. Paste the Identity Provider Single Sign-on URL you copied from Okta into the **Single Sign-On URL** field. 
+      1. For **Identity Provider Signature Certificate**, upload the certificate downloaded from Okta. 
+      1. Select the appropriate method for **Request Binding**. Okta supports HTTP POST and HTTP Redirect.
```
Comment (xugy99) [inline product_docs/docs/biganimal/release/getting_started/identity_provider/okta.mdx:62]: Okta supports all the 3 request binding method: HTTP-Post, HTTP-Redirect, and Hybrid.

File: product_docs/docs/biganimal/release/getting_started/identity_provider/okta.mdx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+   1. On the **Assignments** tab on the Applications page, select **Assign** to assign people or groups to the newly created application. If you need to sign into BigAnimal, be sure to assign yourself.
+   1.  On the **Sign On** tab, select **View SAML setup instructions** to open a tab with instructions for your application. 
+       1. Copy from the Identity Provider Single Sign-on URL from step 1 of the SAML setup instructions.
+       1. Select **Download certificate** in step 3 of the instructions.
+
+   1. In BigAnimal, on the **Setup Config** tab on the Set Up Identity Provider page: 
+      1. Paste the Identity Provider Single Sign-on URL you copied from Okta into the **Single Sign-On URL** field. 
+      1. For **Identity Provider Signature Certificate**, upload the certificate downloaded from Okta. 
+      1. Select the appropriate method for **Request Binding**. Okta supports HTTP POST and HTTP Redirect. 
+      1. For **Response Signature Algorithm**, select **rsa-sha256** .
```
Comment (xugy99) [inline product_docs/docs/biganimal/release/getting_started/identity_provider/okta.mdx:63]: Okta supports both rsa-sha256 and rsh-sha1

File: product_docs/docs/biganimal/release/getting_started/identity_provider/okta.mdx (hunk 3 above)
Comment (drothery-edb) [reply product_docs/docs/biganimal/release/getting_started/identity_provider/okta.mdx:62]: I assume this is how that methods are shown on the UI: HTTP POST, HTTP REDIRECT, and HYBRID. Can you confirm the case and hyphenation?

File: product_docs/docs/biganimal/release/getting_started/identity_provider/okta.mdx (hunk 4 above)
Comment (drothery-edb) [reply product_docs/docs/biganimal/release/getting_started/identity_provider/okta.mdx]: Incorporated

File: product_docs/docs/biganimal/release/getting_started/identity_provider/azure_ad.mdx (hunk 2 above)
Comment (drothery-edb) [reply product_docs/docs/biganimal/release/getting_started/identity_provider/azure_ad.mdx]: Incorporated

File: product_docs/docs/biganimal/release/getting_started/identity_provider/azure_ad.mdx (hunk 1 above)
Comment (xugy99) [reply product_docs/docs/biganimal/release/getting_started/identity_provider/azure_ad.mdx]: yes it is.

image

File: product_docs/docs/biganimal/release/getting_started/identity_provider/okta.mdx (hunk 3 above)
Comment (xugy99) [reply product_docs/docs/biganimal/release/getting_started/identity_provider/okta.mdx]: yes
image

Comment (drothery-edb) [reply product_docs/docs/biganimal/release/getting_started/identity_provider/okta.mdx]: Fixed

File: product_docs/docs/biganimal/release/getting_started/identity_provider/azure_ad.mdx (hunk 1 above)
Comment (drothery-edb) [reply product_docs/docs/biganimal/release/getting_started/identity_provider/azure_ad.mdx]: fixed

**Review by xugy99 — APPROVED** [review]

**Pull request merged**